docs/src/man/split_apply_combine.md

```diff
@@ -778,3 +893,376 @@ julia> df
+   5 │ c                             1
+   6 │ c                             2
+```
+
+Note that this operation also makes sense in a data frame context,
+where all rows are considered to be in the same group:
+
+```jldoctest sac
+julia> transform(df, eachindex)
+6×4 DataFrame
+ Row │ customer_id  transaction_id  volume  eachindex
+     │ String       Int64           Int64   Int64
+─────┼────────────────────────────────────────────────
+   1 │ a                        12       2          1
+   2 │ b                        15       3          2
+   3 │ b                        19       1          3
+   4 │ b                        17       4          4
+   5 │ c                        13       5          5
+   6 │ c                        11       9          6
+```
+
+Finally recall that `eachindex` is a standard function for getting all indices
+in an array. This similarity of functionality was the reason why this name was
+picked:
+
+```jldoctest sac
+julia> collect(eachindex(df.customer_id))
+6-element Vector{Int64}:
+ 1
+ 2
+ 3
+ 4
+ 5
+ 6
+```
+
+This, for example, means that in the following example the two created columns
+have the same contents:
+
+```jldoctest sac
+julia> combine(gdf, eachindex, :customer_id => eachindex)
+6×3 DataFrame
+ Row │ customer_id  eachindex  customer_id_eachindex
+     │ String       Int64      Int64
+─────┼───────────────────────────────────────────────
+   1 │ a                    1                      1
+   2 │ b                    1                      1
+   3 │ b                    2                      2
+   4 │ b                    3                      3
+   5 │ c                    1                      1
+   6 │ c                    2                      2
+```
+
+
+## Column-independent operations versus functions
+
+When discussing column-independent operations it is important to remember
+that operation specification syntax allows you to pass a function (without
+source and target column names), in which case such a function gets passed a
+`SubDataFrame` that represents a group in a `GroupedDataFrame`. Here is an
+example comparing a column-independent operation and a function:
+
+```jldoctest sac
+julia> combine(gdf, eachindex, sdf -> axes(sdf, 1))
+6×3 DataFrame
+ Row │ customer_id  eachindex  x1    
+     │ String       Int64      Int64 
+─────┼───────────────────────────────
+   1 │ a                    1      1
+   2 │ b                    1      1
+   3 │ b                    2      2
+   4 │ b                    3      3
+   5 │ c                    1      1
+   6 │ c                    2      2
+```
+
+Notice that column independent operation `eachindex` produces the same result
+as using anonymous function `sdf -> axes(sdf, 1)` that takes a `SubDataFrame`
+as its first argument and returns indices along its first axes.
+Importantly without special definition of column-independent operation
+the `eachindex` function would fail when being passed as you can see here:
```

Comment:
```suggestion
Notice that the column independent operation `eachindex` produces the same result
as using the anonymous function `sdf -> axes(sdf, 1)` that takes a `SubDataFrame`
as its first argument and returns indices along its first axes.
Importantly if it wasn't defined as a column-independent operation
the `eachindex` function would fail when being passed as you can see here:
```